common/components/controllers/AboutUserController.jsx

```diff
@@ -92,15 +133,20 @@ class AboutUserController extends React.PureComponent<{||}, State> {
   _renderRightColumn(user: UserAPIData): React$Node {
     return (
       <React.Fragment>
-        {user.about_me && this._renderAboutMe(user)}
+        {(user.about_me || this.state.isUserOrAdmin) &&
+          this._renderAboutMe(user)}
 
-        {user && !_.isEmpty(user.user_technologies)
+        {user &&
+        (!_.isEmpty(user.user_technologies) || this.state.isUserOrAdmin)
           ? this._renderAreasOfInterest(user)
           : null}
 
-        {user && !_.isEmpty(user.user_files) ? (
+        {!_.isEmpty(user.user_files) || this.state.isUserOrAdmin ? (
           <div className="about-user-section">
-            <h2 className="text-uppercase">Files</h2>
+            <span className="side-by-side">
+              <h2 className="text-uppercase">Files</h2>
```

Comment:
No other header on this page uses `text-uppercase` anymore, so let's remove it from here.